0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -141,15 +187,25 @@ The response body is a JSON object that includes basic account details necessary
 | `receiver_info.name` | String | _(OPTIONAL)_ Full name of the individual, company or organization the receiver represents |
 | `receiver_info.image_url` | HTTPS URL | _(OPTIONAL)_ URL where the sender can get a picture representation of the receiver |
+| `asset_info` | Object | _(OPTIONAL)_ Details about the server's asset, for display purposes. |
+| `asset_info.code` | String | Asset code to identify the server's currency. Currencies that have [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codes should use those. Sender UIs SHOULD be able to render non-standard codes |
+| `asset_info.scale` | Integer | The scale of the amounts on the server's account (e.g. an amount of `"1000"` with a scale of `2` translates to `10.00` units of the server's asset/currency) |
+| `pull_balance` | Object |	_(OPTIONAL)_ Monetary details of a pull payment token. Used only for pull payments. |
+| `pull_balance.available` |	Integer String |	Amount, denoted in the minimum divisible units of the server's account, that can currently be pulled from the account. |
+| `pull_balance.refill_maximum` |	Integer String |	Amount, denoted in the minimum divisible units of the server's account, that `balance.available` will be filled up to at `pull_info.refill_time`. |
+| `pull_info`	|	Object | _(OPTIONAL)_ Non-monetary details of a pull payment token. Used only for pull payments. |
+| `pull_info.start_time` | String |	[ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) UTC timestamp, e.g. "2019-01-10T01:01:13Z", representing the time at which `pull_balance.available` will be filled up to the amount of `pull_balance.refill_maximum` for the first time. |
+| `pull_info.expiry_time` |	String | _(OPTIONAL)_ [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) UTC timestamp, e.g. "2019-07-10T01:01:12Z", representing the time after `which pull_balance.available` will not be filled up anymore. |
+| `pull_info.refill_time` |	String | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) UTC timestamp, e.g. "2019-02-10T01:01:13Z", representing the time at which `pull_balance.available` will be filled up to the amount of `pull_balance.refill_maximum` for the next time.
+| `pull_info.frequency`	| String | Frequency at which `pull_info.refill_time` is incremented. Possible values are `DAY`, `WEEK`, `MONTH`, `YEAR`. |
+| `pull_info.frequency_interval` | Integer | Interval associated with `pull_info.frequency`. For example, if `pull_info.frequency` is `WEEK` and `pull_info.frequency_interval` is `2`, `pull_balance.available` is refilled every 2 weeks. 
+| `pull_info.asset_code` | String	| Asset code to identify the currency agreed upon for this pull payment. Currencies that have [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codes should use those. |
```

Comment:
I thought about that but was not sure. I think you still need the "external" `asset_info` field because this describes the currency of the server whereas `pull_info.asset_code` is the currency that the sender and the receiver agreed upon when they negotiated the terms of the pull payment token. However, it does make sense to include the asset_scale unless everything is in the minimum divisible unit of that currency. 
I just realized that I have to change the descriptions in `pull_balance` to not say "units of the server's account" but "units of the agreed upon currency".